### PR TITLE
Add missing stripped StateEvents

### PR DESCRIPTION
**Pull request by mujx**
Fixes deserialization issues from ruma/ruma#127

### CODE REVIEW

File: src/stripped.rs [hunk 1]
```diff
@@ -1,27 +1,60 @@
...
 
 use EventType;
+use room::aliases::AliasesEventContent;
 use room::avatar::AvatarEventContent;
 use room::canonical_alias::CanonicalAliasEventContent;
-use room::join_rules::JoinRulesEventContent;
+use room::create::CreateEventContent;
+use room::guest_access::GuestAccessEventContent;
+use room::history_visibility::HistoryVisibilityEventContent;
+use room::join_rules::*;
```
Comment (jimmycuadra) [inline src/stripped.rs]: Use explicit imports here, please!

**Comment by jimmycuadra** [comment]
Looks good, just one minor change to get rid of the glob import. Thanks!

**Comment by mujx** [comment]
I used the glob to avoid the unused  import warning for `JoinRule` but I fixed it with `super::super`.

**Comment by jimmycuadra** [comment]
I'd just use the regular import path in the tests rather than `super`, for items that aren't actually defined in the parent module. That is, only `StrippedRoomTopic` and `StrippedState` would be imported using `super`.

**Pull request merged**